code/modules/mob/living/living_healthscan.dm

```diff
@@ -87,6 +93,7 @@ GLOBAL_LIST_INIT(known_implants, subtypesof(/obj/item/implant))
 
 /datum/health_scan/ui_data(mob/user, data_detail_level = null)
 	var/list/data = list(
+		"patient_mob" = WEAKREF(target_mob), // this doesn't work just yet
```

Comment:
Does it work yet?

Reply:
oops, I left this in, apparently I didn't need a ref to the mob because the datum itself already had it stored